rq/worker.py

```diff
@@ -699,6 +728,7 @@ def bootstrap(
         self.subscribe()
         self.set_state(WorkerStatus.STARTED)
         qnames = self.queue_names()
+        self._dequeue_strategy = dequeue_strategy
```

Comment:
Sorry, one hopefully last review, `dequeue_strategy` doesn't need to be passed into `bootstrap()` as we're not doing anything with this here. We can simply assign `dequeue_strategy` to `self` inside `work()`.

Reply:
No worries, we discussed a bit of this [here](https://github.com/rq/rq/pull/1806#discussion_r1111508285) and [here](https://github.com/rq/rq/pull/1806#discussion_r1111553095), but you didn't follow up. I'll make the change then.